src/less/donut-chart/donut-chart.less

```diff
@@ -0,0 +1,185 @@
+    width: 100%;
+}
+.donut-chart__container-vertical {
+    margin-bottom: 20px;
+    max-width: 200px;
+    width: 100%;
+}
+.donut-chart-legend__container {
+    display: inline-block;
+    width: 100%;
+}
+.donut-chart__stacked {
+    max-width: 190px;
+}
+.donut-chart__legend-left {
+    flex-flow: row-reverse nowrap;
+    justify-content: flex-start;
+    max-width: 450px;
+}
+.donut-chart__legend-left .donut-chart__container {
+    margin-left: 30px;
+    max-width: 190px;
+}
+.donut-chart__legend-right {
+    flex-flow: row nowrap;
+    justify-content: flex-start;
+    max-width: 450px;
+}
+.donut-chart__legend-right .donut-chart__container {
+    margin-right: 30px;
+    max-width: 190px;
+}
+.donut-chart__legend-right .donut-chart-legend__container {
+    max-width: 210px;
+    width: 50%;
+}
+.donut-chart__legend-left .donut-chart-legend__container {
+    max-width: 210px;
+    width: 50%;
+}
+.donut-chart svg {
+    display: inline-block;
+    position: relative;
+    vertical-align: middle;
+    width: 100%;
+}
+.donut-chart__metrics {
+    display: inline-block;
+    width: 50%;
+}
+.donut-chart__legend-left .donut-chart__metrics {
+    width: 100%;
+}
+.donut-chart__legend-right .donut-chart__metrics {
+    width: 100%;
+}
+.donut-chart__stacked .donut-chart__metrics {
+    display: block;
+    margin-bottom: 30px;
+    width: 100%;
+}
+.donut-chart__horizontal-metrics {
+    margin-bottom: 20px;
+}
+.donut-chart__metric {
+    display: block;
+    font-size: 30px;
+    font-weight: 700;
+    line-height: 40px;
+}
+.donut-chart__subtitle {
+    display: block;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 18px;
+}
+.donut-chart__segment {
+    opacity: 1;
+    transition: opacity 0.3s;
+}
+.donut-chart__segment--primary {
+    fill: var(--data-viz-primary, var(--color-data-viz-chart-primary));
+    stroke: var(--data-viz-primary, var(--color-data-viz-chart-primary));
+}
+.donut-chart__segment--secondary {
+    fill: var(--data-viz-secondary, var(--color-data-viz-chart-secondary));
+    stroke: var(--data-viz-secondary, var(--color-data-viz-chart-secondary));
+}
+.donut-chart__segment--tertiary {
+    fill: url("#dataVizPattern0");
+    stroke: var(--color-data-viz-chart-quaternary-stroke);
+}
+.donut-chart__segment--quaternary {
+    fill: url("#dataVizPattern1");
+    stroke: var(--color-data-viz-chart-tertiary-stroke);
+}
+.donut-chart__segment--quinary {
+    fill: var(--data-viz-quinary-background, var(--color-data-viz-chart-quinary-background));
+    stroke: var(--data-viz-quinary-stroke, var(--color-data-viz-chart-quinary-stroke));
+}
+.donut-chart--inactive {
+    opacity: 0.2;
+}
+.donut-chart__tooltip {
+    color: var(--color-neutral-7);
+    display: block;
+    font-size: 12px;
+    pointer-events: none;
+    position: fixed;
+}
+.donut-chart__tooltip__label {
+    display: block;
+    font-weight: 700;
+}
+.donut-chart__tooltip__value {
+    display: block;
+}
+.donut-chart__tooltip-container {
```

Comment:
Lets reuse this if we can: https://github.com/eBay/skin/blob/master/src/less/tooltip/tooltip.less#L13-L20